base/applications/taskmgr/applpage.c

```diff
@@ -944,6 +944,8 @@ void ApplicationPage_OnGotoProcess(void)
         i = ProcGetIndexByProcessId(dwProcessId);
         if (i != -1)
         {
+            ListView_SetSelectionMark(hProcessPageListCtrl, i);
```

Comment:
How does this differ with the `SetItemState` from below? (is there also some sort of visual difference, with and without that line?)

Reply:
Yes.

Reply:
I cannot reproduce this problem on unpatched Task Manager in Windows 10. Need to check on XP/2003.